src/SignhostAPIClient.Tests/SignHostApiClient.cs

```diff
@@ -19,11 +20,58 @@ public class SignHostApiClientTests
 			Endpoint = "http://localhost/api/"
 		};
 
+        [Fact]
+        public void calculated_valid_checksum_results_in_no_exception() {
+            //Arrange
+            var mockHttp = new MockHttpMessageHandler();
+            var validChecksum = "0f628893e173dce50c89238493e3a21a61267b21";
+            var transactionId = "81eef8bf-0661-4f6f-aa2f-5cb506cdc065";
+            var transactionStatus = "10";
+            var sharedSecret = "UwMKSuFPtydaZHpNzVyJDfgX7f1wMf6F";
+
+
+            using (var httpClient = mockHttp.ToHttpClient())
+            {
+                var signhostApiClient = new SignHostApiClient(settings, httpClient);
+
+                //Act
+                Action x = () => signhostApiClient.CalculateAndValidateChecksum(transactionId, transactionStatus, sharedSecret, validChecksum);
+
+                //Assert
+                mockHttp.VerifyNoOutstandingRequest();
+                x.ShouldNotThrow<ErrorHandling.BadAuthorizationException>();
+            }
+        }
+
+        [Fact]
+        public void calculated_invalid_checksum_results_in_BadAuthorizationException() {
+        
```

Comment:
No newline

Reply:
Additionally the `{` should be on its own line here.